app/models/integration.rb

```diff
@@ -0,0 +1,42 @@
+require 'yaml'
+
+class Integration
+  INTEGRATIONS = YAML.load File.read(Rails.root.join('data', 'integrations.yml')).freeze
+  EVENT_TYPES = YAML.load File.read(Rails.root.join('data', 'event_types.yml')).freeze
+
+  # NOTE: Temporarily obscure integrations that aren't ready yet
+  INCOMPLETE_INTEGRATIONS = %w(hip_chat jira pivotal_tracker).freeze
+
+  class NotFound < StandardError
+  end
+
+  def self.find key
+    data = INTEGRATIONS.fetch(key) do
+      raise NotFound, %(Integration "#{key}" is not supported)
+    end
+
+    { 'key' => key }.merge data
+  end
+
+  def self.exists? key
+    INTEGRATIONS.key? key
+  end
+
+  def self.keys
+    INTEGRATIONS.keys.freeze
+  end
+
+  def self.supported_integrations
+    INTEGRATIONS.map do |key, data|
+      { 'key' => key }.merge data
+    end
+  end
+
+  def self.public_integrations
+    supported_integrations.reject { |int| INCOMPLETE_INTEGRATIONS.include?(int['key']) }
```

Comment:
This is just a temp hack so that the Rainforest app doesn't show integrations that aren't done yet. We still need supported_integrations to include all of them though, or else the specs would break.

This will go away once Jira, HipChat, and Pivotal are done.